vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -146,9 +147,15 @@ func (calc *UpdatePriorityCalculator) getUpdatePriority(pod *apiv1.Pod, recommen
 	for _, podContainer := range pod.Spec.Containers {
 		recommendedRequest := vpa_api_util.GetRecommendationForContainer(podContainer.Name, recommendation)
 		if recommendedRequest == nil {
+			if strings.HasPrefix(pod.Name, "prometheus") {
```

Comment:
Looks like this makes sense to log for other pods as well or is it too much?

Reply:
Nevermind. This looks like a temporary hack :smiley:

Reply:
I suggest we do it for all pods because it can happen to other than Prometheus. Last time it was for example observed for kube-downscaler.

Reply:
@linki exactly temporary hack to understand better. 
@mikkeloscar I don't know how much it will log, so I decided for only the one important one.
Honestly downscaler is so minor I would not waste my time looking into its issue.